functional-test/src/test/groovy/org/rundeck/util/gui/pages/project/ProjectEditPage.groovy

```diff
@@ -216,7 +216,11 @@ class ProjectEditPage extends BasePage {
      * @param replacement
      */
     def replaceConfiguration(String original, String replacement){
-        ((JavascriptExecutor) context.driver).executeScript("ace.edit('_id0').session.replace(ace.edit('_id0').find('${original}',{wrap: true, wholeWord: true }), '${replacement}');")
+        try {
```

Comment:
Looks pretty fragile and hacky but my lack of knowledge in this area prevents me from offering an alternative solution. If you believe there is no better alternative, I will approve it, but I would urge to give it another thought.